.github/workflows/mergequeue_docs.yml

```diff
@@ -0,0 +1,34 @@
+# Licensed under the Apache License, Version 2.0 or the MIT License.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Copyright Tock Contributors 2023.
+
+# Netlify's own CI builds both deploy previews for PRs, as well as the
+# production deploy for the master branch. We use this workflow purely as we
+# can't have Netlify build the wildcard gh-readonly-queue/* branches. This
+# workflow thus ensures that docs need to build (albeit not in the exact same
+# envrionment as Netlify's).
```

Comment:
This is a sentence fragment. I think you meant to say "need to build do build" where you wrote "need to build".

Reply:
Ah, yes, I should've phrased this more clearly. What I meant to say was "[...] thus ensures that the docs build successfully".